### DISCUSSION TITLE
Copilot not suggesting VS2022

### DISCUSSION THREAD

**Discussion by Lilyy2565** [community/community#30842] [Copilot Conversations]
When I am trying to use copilot even for something simple like adding 2 numbers nothing happens.
![image](https://user-images.githubusercontent.com/52353209/186750879-5586bf15-851f-4e02-843d-f7cf8e4d90ec.png)
I have tried the following:
- Globally enabling and disabling copilot
- Reinstalling copilot
- Signing in and out of copilot
- Restarting visual studio multiple times

The output logs for `GitHub Copilot` do not show any errors and there are no errors (that I know of) while authenticating.

**Comment by Bortronx** [comment]
unacceptable for a paid plan when there is a blog post advertising this feature.

**Comment by davidegiacometti** [comment]
Have you tried to delete `%LOCALAPPDATA%\github-copilot`?

**Reply by Lilyy2565** [reply]
Didn't seem to work, just wanted to be unauthenticated.
Tried `// Add two numbers` again and it still isn't suggesting anything.

**Reply by Matt-17** [reply]
There is only on file `hosts.json` containing the login token
